docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -3,42 +3,88 @@ title: Set Up Code Builder
 lang: en
 ---
 
-## Install the Code Builder Managed Package
+Enable Code Builder to provide the permissions needed to install the Code Builder managed package in a Salesforce supported org.
 
-Salesforce Code Builder is a second-generation (2GP) managed package that a user with requisite permissions can install in any supported Salesforce org edition.
+### Required Editions
 
-1.  Click the install [link](https://appexchange.salesforce.com/appxListingDetail?listingId=a0N4V00000HddeKUAR) to access the Code Builder managed package on AppExchange.
-2. Click **Get It Now**.
-3. Select the org in which you want to install Code Builder.
-4. Check terms and conditions and click **Confirm and Install**.
+**Available in**: Lightning Experience in Enterprise, Performance, Professional and Unlimited Editions.
```

Comment:
```suggestion
**Available in**: Lightning Experience in Enterprise, Performance, Professional, and Unlimited Editions.
```

Reply:
I assume that Code Builder requires API access, just like most of DX tooling. We have this little blurb for DOCe. Should it be added, @smaddox-sf? 
To install DevOps Center in a Professional Edition org, the org must have API access, which enables DevOps Center to access data from the source control system. If you attempt to install DevOps Center in a Professional Edition org without API access, an installation error occurs. Contact your Account Executive to request the API add-on.

Reply:
I deleted my comment about API access where I tagged @smaddox-sf. Sorry, I now see it below.